src/lib/modules/Document/DocumentAuthors.js

```diff
@@ -246,6 +255,7 @@ DocumentAuthors.defaultProps = {
   withPopUpShowMoreFields: false,
   showAllFieldsInPopUp: true,
   listItemAs: null,
+  noClassName: false,
```

Comment:
I am not sure I understand very well the meaning of `noClassName: false`. Is it to have a default?

Reply:
I'm also curious

Reply:
Yes, most of the times this component is used it needs the className `document-authors-list-wrapper`, I added this prop for the cases where we don't want that className (only 2 cases). (Did it this way to prevent passing the same className all the time we use this component, but can change it to `className` and use it)

Reply:
thank you for the explanation! In this case, can we maybe improve the naming of the prop?
If we use something like `withListWrapper` it might be more clear. I would choose a name that explain what it does: it looks like that is setting a max height and adding a scroll? Some options:
* `withVerticalScroll`
* `withMaxSize`
any other idea?

Reply:
`withVerticalScroll` seems good for me.